lib/acts-as-messageable/scopes.rb

```diff
@@ -28,8 +28,8 @@ def initialize_scopes
                                                   :r_perm_delete  => false,
                                                   :s_perm_delete  => false)
         }
-        scope :readed,            lambda { where(:opened => true)  }
-        scope :unreaded,          lambda { where(:opened => false) }
+        scope :readed,            lambda { where('opened_at is not null') }
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.